cfgov/v1/documents.py

```diff
@@ -12,11 +18,25 @@
 from v1.models.newsroom_page import LegacyNewsroomPage, NewsroomPage
 
 
+ngram_tokenizer = analyzer(
+    'filterable_ngram_tokenizer',
+    tokenizer=tokenizer(
+        'filterable_tokenizer',
+        'ngram',
+        min_gram=1,
+        max_gram=16,
+        token_chars=["letter", "digit", "symbol"]
+    ),
+    filter=['lowercase', token_filter('ascii_fold', 'asciifolding')]
+)
```

Comment:
Could or should this live in `search.elasticsearch_helpers` along side the existing `ngram_tokenizer` we have there, with naming/commentary on what's different between them?

Reply:
I debated that - I initially left it here as I didn't have a good plan for explaining the difference between the two so I felt grouping this analyzer with its usage made more sense. I guess the main difference would be the size of the tokens, as well as the addition of symbols to this analyzer.

Reply:
Ended up not using the analyzer so deleting it - using a match_phrase_prefix query now.